hands_on/pull_remote.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+from exercise_utils.cli import run_command
+from exercise_utils.git import clone
+
+__requires_git__ = True
+__requires_github__ = True
+
+
+def download(verbose: bool):
+    os.makedirs("samplerepo-finances")
+    clone("https://github.com/git-mastery/samplerepo-finances.git", verbose) 
+    os.chdir("samplerepo-finances")
+    run_command(["git", "remote", "set-url", "origin", "https://github.com/git-mastery/samplerepo-finances-2.git"], verbose) 
```

Comment:
Also curious what the rationale behind abstracting `clone` to a utility function but not `set-url`?